mllib/src/main/scala/org/apache/spark/mllib/linalg/Vectors.scala

```diff
@@ -712,6 +722,9 @@ class SparseVector @Since("1.0.0") (
     }
   }
 
+  @Since("1.6.0")
+  private[spark] def foreach(f: (Double) => Unit) = { indices.foreach { i: Int => f(values(i)) } }
```

Comment:
Yeah, it's `foreachValue`, but you can already operate on all 'active' values with `foreachActive`. Does this add much/